client/job.lua

```diff
@@ -86,10 +86,11 @@ RegisterNetEvent('QBCore:Client:OnPlayerLoaded', function()
     local player = PlayerId()
     CreateThread(function()
         Wait(5000)
-        SetEntityMaxHealth(ped, 200)
-        SetEntityHealth(ped, 200)
-        SetPlayerHealthRechargeMultiplier(player, 0.0)
-        SetPlayerHealthRechargeLimit(player, 0.0)
+        QBCore.Functions.GetPlayerData(function(PlayerData)
```

Comment:
Could you remove this thread and move this code down into the other thread, that's a thread less then because I don't see the usage of this one thread before